cognee/api/v1/visualize/visualize.py

```
@@ -3,10 +3,12 @@
 )
 from cognee.infrastructure.databases.graph import get_graph_engine
 import logging
+import structlog
+from cognee.shared.logging_utils import setup_logging
 
 
 import asyncio
-from cognee.shared.utils import setup_logging
+from cognee.shared.logging_utils import setup_logging
```

Ruff (0.8.2)

Redefinition of unused `setup_logging` from line 7

Remove definition: `setup_logging`

(F811)